prow/plugins/size/test-fixtures/k8s-docs-.generated_docs.paths

```diff
@@ -0,0 +1,5 @@
+# first 30 lines of kubernetes/docs/.generated_docs
```

Comment:
In go it's normal to call this folder `testdata` instead of `test-fixtures`. Various go tools are smart about that name.